lib/rules/disallow-space-after-object-keys.js

```diff
@@ -7,6 +7,7 @@
  *  - `true`
  *  - `"ignoreSingleLine"` ignores objects if the object only takes up a single line
  *  - `"ignoreMultiLine"` ignores objects if the object takes up multiple lines
+ *  - `"ignoreAligned"` ignores aligned object properties
```

Comment:
Please see the latest commit. I also added checks to prevent combination of `multiline` with either `singleline` or `aligned`, doesn't seem to make sense.